KinSDK/KinSDK/KinClient.swift

```diff
@@ -13,48 +13,28 @@ import Foundation
  */
 public final class KinClient {
     /**
-     Convenience initializer to instantiate a `KinClient` with a `ServiceProvider`.
+     Instantiates a `KinClient` with a `Network` and an `AppId`.
 
-     - Parameter provider: The `ServiceProvider` instance that provides the `URL` and `Network`.
-     - Parameter appId: The `AppId` of the host application.
-     */
-    public convenience init(provider: ServiceProvider, appId: AppId) {
-        self.init(with: provider.url, network: provider.network, appId: appId)
-    }
-
-    /**
-     Instantiates a `KinClient` with a `URL` and a `Network`.
-
-     - Parameter nodeProviderUrl: The `URL` of the node this client will communicate to.
      - Parameter network: The `Network` to be used.
      - Parameter appId: The `AppId` of the host application.
      */
-    public init(with nodeProviderUrl: URL, network: Network, appId: AppId) {
```

Comment:
@yosriz another change to unify the sdks is removing the network function bubble in ios. using a shared singleton now like android.